test/release/examples/primers/classes.chpl

```diff
@@ -117,9 +122,9 @@ var b1 = own.borrow();
 // A class type without a decorator, such as ``C``, is the same
 // as ``borrowed C``. The ``this`` argument of a method is a borrow as well.
 
-// The compiler will automatically converts from ``owned`` ``shared``
-// or ``unmanaged`` in the process of doing a method or function call
-// or variable initialization.
+// The compiler automatically adds conversion from ``owned`` ``shared``
```

Comment:
I think there should be a comma between ``owned`` and ``shared``, not sure how I missed that the first time XD